windows/src/README.md

```diff
@@ -151,16 +151,14 @@ executables when you build a release.
 To include UserDefines.mak in the build, use the command line parameter `-DUSERDEFINES`. You
 can also set an environment variable `USERDEFINES=1` to get the same result.
 
-### KEYMAN_CEF4DELPHI_ROOT - Chromium Embedded Framework in Keyman Developer
+### KEYMAN_CEF4DELPHI_ROOT - Chromium Embedded Framework
 
-Keyman Developer uses Chromium Embedded Framework. The source repo is at
+Keyman and Keyman Developer use Chromium Embedded Framework. The source repo is at
```

Comment:
Maybe "Keyman for Windows and Keyman Developer use Chromium..."

Reply:
Not inclined to worry about these